### ISSUE TITLE
Cannot set accelerator for actions because signature is incorrect

### ISSUE THREAD

**Issue by Kangaroux** [GtkSharp/GtkSharp#165]
https://developer.gnome.org/gio/stable/GActionGroup.html#g-action-group-list-actions

Returns a NULL-terminated array of the names of the actions in the group. 

```C#
public interface IActionGroup : IWrapper
{
    // ...
    string ListActions();
    // ...
}
```

**Comment by Kangaroux** [comment]
This issue also appears for methods like `Gtk.Application.SetAccelsForAction`, `Gtk.Application.GetAccelsForAction`, and `Gtk.Application.GetActionsForAccel`, to name a few.

**Comment by zii-dmg** [comment]
Second part is already fixed by #145.
